manifests/m/MDFtoISO/MDFtoISO/1.0/MDFtoISO.MDFtoISO.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MDFtoISO.MDFtoISO
+PackageVersion: 1.0
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: http://www.softsea.com/download.php?id=832996351
+  InstallerSha256: 216D4BCEA4BBE226F0C610C59B911CF9351FEA34178A1F8EC04DAD99F4F1BDBC

```

Comment:
```suggestion
  InstallerSha256: 7087A6A80F326F6AC8D839BAA14029BD101DB5213F1FC2BBD3F984F6901B098B
```

Reply:
This is to revert to the original hash. The multiple download links on the page confused me and I selected the wrong package yesterday. I do not know what hash is being received by the validation pipeline, but I can manually validate this with the reverted hash.